### ISSUE TITLE
SYS_PTRACE question

### ISSUE THREAD

**Issue by virtuald** [andyneff/wine_msys64#2]
First, this looks cool, and a good step towards what I want (being able to build windows stuff that depends on msys2 in wine + docker).

Just curious, what is it about msys2 that requires SYS_PTRACE?

**Comment by andyneff** [comment]
@virtuald That's a very good question. I'm not 100% sure what's going on, but if you don't do it, you get the following error

```
err:module:attach_process_dlls "msys-2.0.dll" failed to initialize, aborting
err:module:LdrInitializeThunk Main exe initialization for L"C:\\msys64\\usr\\bin\\mintty.exe" failed, status c0000005
```

So I'm guessing that for some reason, loading a dll actually uses PTRACE.

**Comment by TungLove** [comment]
Duplicate of #
